ui-admin/src/components/InfoCard.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { isEmpty } from 'lodash'
+
+
+/**
+ * Stylized card component for displaying read-only or editable data.
+ */
+export function InfoCard({ children }: { children: React.ReactNode }) {
```

Comment:
I like this style of card that erin seems to use in multiple places in the designs. I decided to make it a fully generic component so that hopefully we can use it across the app.

I'm not sure if `InfoCard` is the best name for it, but it's the best I could come up with. Let me know if you have any ideas.